test/io/jesi/backpack/test/strict_test.cljc

```diff
@@ -163,7 +163,7 @@
       (is (= (is (= 1 1))
              (strict/is= 1 1))))))
 
-(strict/deftest -deftest true)
+(strict/deftest ^:focus -deftest true)
```

Comment:
remove focus

Reply:
Maybe change the meta to something else if you want to ensure the meta is preserved?